tests/unittest/_torch/attention/sparse/conftest.py

```
@@ -0,0 +1,136 @@
+import pytest
+import torch
+
+
+def _tie_aware_check_impl(
+    out_indices: torch.Tensor,
+    logits: torch.Tensor,
+    seq_lens: torch.Tensor,
+    top_k: int,
+    next_n: int,
+    compress_ratio: int = 1,
+    ref_vals_cache: dict = None,
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)